.circleci/config.yml

```diff
@@ -582,7 +582,7 @@ jobs:
   run-test-ios-15:
     <<: *base-job
     # Fix-me: running on M1 makes these tests crash
-    resource_class: macos.x86.medium.gen2
+    resource_class: macos.m1.medium.gen1
```

Comment:
this is already the resource _class in the `base-job` so I think we can just remove this?